spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -151,9 +153,12 @@ public TransactionInfo getTransactionInfo() {
                 case RollbackFailure:
                     failureHandler.onRollbackFailure(e.getTransaction(), e.getCause());
                     throw e.getCause();
+                case ParticipantRollbackDone:
+                    throw new TransactionException(TransactionExceptionCode.ParticipantReportedRollback,e.getOriginalException());
+                case ParticipantRollbackFailure:
+                    throw e.getCause();
```

Comment:
A->B->C->D->E
if E throw e.getCause(), D, B, C, and A may all request a rollback with the TC. 
It is recommended that the participant fails to roll back, and the upstream link no longer requests the tc rollback.